composeApp/src/commonMain/kotlin/com/github/familyvault/ui/screens/main/chat/SelectChatContent.kt

```diff
@@ -54,6 +55,16 @@ fun SelectChatContent() {
             }
         }
 
+        chatThreadListenerService.startListeningForUpdatedChatThread {
+            currentChatThreadsState.clean()
```

Comment:
Dlaczego nie możemy po prostu usunąć jednego elementu który został usunięty i dodać go zaaktualizowanego. Zobacz jak to zrobiłem w `editExistingChatThreadLastMessage`.